README.md

```diff
@@ -1,78 +1,110 @@
 # osb-checker
+
 An automatic checker to verify an Open Service Broker API implementation against the [specification](https://github.com/openservicebrokerapi/servicebroker).
 
-## Project Status
+# Project Status
 
 This project should be considered **experimental**. You should validate the results against the released [specification](https://github.com/openservicebrokerapi/servicebroker). In the case of any discrepancy, the specification should be considered correct.
 
-## Usage - test against the mock OSB API endpoint
+# Usage
+
+## The Mock OSBAPI Server
+
+This repository comes with a mock OSBAPI compatible server. Follow the steps
+below to start the mock server and run tests against it.
+
+1. Run `git clone` to clone this repository to `$DIRECTORY`
 
-1. Clone the repository to a folder, for example c:\osb-checker
 2. Launch the mock server
+
 ```bash
-    cd c:\osb-checker\2.13\mocks
-    npm install
-    node mockOSB.js
+cd $DIRECTORY/$VERSION_NUMBER/mocks
+npm install
+node mockOSB.js
 ```
-3. Run tests
+
+3. In a new terminal window, run the tests
+
 ```bash
-    cd c:\osb-checker\2.13\tests
-    npm install
-    npm test
+cd $DIRECTORY/$VERSION_NUMBER/tests
```

Comment:
I think this should be changed to `cd $DIRECTORY/$VERSION_NUMBER/tests/test`

Reply:
Good catch! Will fix.